mathics/builtin/graphics.py

```diff
@@ -660,32 +730,45 @@ class ColorDistance(Builtin):
 
     CIE76: euclidean distance in the LABColor space
     CIE94: euclidean distance in the LCHColor space
+    CIE2000 or CIEDE2000: CIE94 distance with corrections
+    CMC: Colour Measurement Committee metric (1984)
     DeltaL: difference in the L component of LCHColor
     DeltaC: difference in the C component of LCHColor
     DeltaH: difference in the H component of LCHColor
 
+    It is also possible to specify a custom distance
+
     >> N[ColorDistance[Magenta, Green], 5]
      = 2.2507
     """
```

Comment:
There's [this](http://colormine.org/delta-e-calculator/cie2000) site for CIE2000. More, we can always check the values against MMA - the output is OK.